client/components/Layout.js

```diff
@@ -1,36 +1,46 @@
 import React from "react";
 import {Layout} from "antd";
-
```

Comment:
I think it would be better to leave this newline here since it distinguishes between module and local files `require`s. It would be great if you could add it to all files which don’t currently have such newline as well.

Reply:
Will fix it,